PSReadLine.build.ps1

```diff
@@ -490,11 +310,10 @@ task Publish -If ($Configuration -eq 'Release') {
 Synopsis: Remove temporary items.
 #>
 task Clean {
-    git clean -fdx
+    git clean -fdX
```

Comment:
This doesn't really guarantee a clean build now that `dotnet` proj files don't require listing each source file.

Reply:
Using `-x` will remove all untracked files, meaning that manually created files (intentionally) will be removed as well if you forget to stage them.
`-X` remove only files ignored by `.gitignore`, and keep the manually created files.

Reply:
I understand that you could lose work, but you could also produce a build (and say publish) that does not reflect the repo because you include untracked files.

Maybe it's reasonable to drop `-f` to before deleting untracked files.

Reply:
Sure, I will change that.

Reply:
`clean.requireForce` defaults to true, so `-dx` doesn't do anything unless `f` is specified. I will just revert this change.